### PR TITLE
[9.0][ADD] account_payment_order_return

### PR DESCRIPTION
**Pull request by luismontalba**
Account Payment Order Return
=========================
This module adds a check in the wizard where move lines are imported into
payment order.
This check lets avoid selecting lines from invoices linked to a payment return.

Usage
--------

* Go to Invoicing > Payments > Debit Orders.
* Create a new record.
* Click on button "Create Payment Lines fron Journal Items" to open the
   wizard.
* Click on the "Linked to an Invoice or Refund " check.
* Click on the "Avoid move lines from returns " check.
* Fill other options.
* Click on button "Add All Move Lines".

cc @Tecnativa

### CODE REVIEW

File: account_payment_order_return/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - Luis M. Ontalba
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    'name': 'Account Payment Order Return',
+    'version': '9.0.1.0.0',
+    'category': 'Project Management',
```
Comment (sergio-teruel) [inline account_payment_order_return/__openerp__.py]: Change category

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Comment by sergio-teruel** [comment]
I con not try in runbot... i think now is very busy jeje... OCA Codesprint...
But code review ok

File: account_payment_order_return/__openerp__.py (hunk 1 above)
Comment (luismontalba) [reply account_payment_order_return/__openerp__.py]: Category changed

**Comment by luismontalba** [comment]
Runbot OK again !

File: account_payment_order_return/wizard/account_payment_line_create.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# Copyright 2017 Tecnativa - Luis M. Ontalba
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import api, fields, models
+
+
+class AccountPaymentLineCreate(models.TransientModel):
+    _inherit = 'account.payment.line.create'
+
+    avoid_returned = fields.Boolean(
```
Comment (pedrobaeza) [inline account_payment_order_return/wizard/account_payment_line_create.py]: Make the contrary logic: put a field called `include_returned` and unless this is checked, you include in the domain the new condition.

Comment (luismontalba) [reply account_payment_order_return/wizard/account_payment_line_create.py]: Done

**Review by pedrobaeza — APPROVED** [review]
@cubells @chienandalu please review

**Review by chienandalu — APPROVED** [review]
Tested on runbot :+1:

**Comment by pedrobaeza** [comment]
Please migrate it to v10/v11

**Pull request merged**